src/BuildScriptGenerator/Php/PhpScriptGeneratorServiceCollectionExtensions.cs

```diff
@@ -16,7 +15,6 @@ public static IServiceCollection AddPhpScriptGeneratorServices(this IServiceColl
         {
             services.TryAddEnumerable(ServiceDescriptor.Singleton<ILanguageDetector, PhpLanguageDetector>());
             services.TryAddEnumerable(ServiceDescriptor.Singleton<IProgrammingPlatform, PhpPlatform>());
-            services.TryAddEnumerable(ServiceDescriptor.Singleton<IConfigureOptions<PhpScriptGeneratorOptions>, PhpScriptGeneratorOptionsSetup>());
             services.AddSingleton<IPhpVersionProvider, PhpVersionProvider>();
             services.AddScoped<PhpLanguageDetector>();
```

Comment:
nit:this can be singleton too?

Reply:
yes, thanks. I am sending a cleanup PR having fixes for this.